README.md

```diff
@@ -101,13 +101,14 @@ export TEAM=<optional - your team name ; used for cost allocation purposes>
 export PROJECT=<optional - your project name ; used for cost allocation purposes>
 ```
 
-5. Set up Helm and add the `stable` Helm Charts repository
+5. Set up Helm and add the required chart repositories
 
 If you haven't, [install helm now](https://helm.sh/docs/intro/install/).
 
 ```sh
 $ helm repo add stable https://kubernetes-charts.storage.googleapis.com/
 $ helm repo add bitnami https://charts.bitnami.com/bitnami
+$ helm repo add influxdata https://influxdata.github.io/helm-charts
```

Comment:
```suggestion
$ helm repo add influxdata https://helm.influxdata.com/
```

Reply:
I guess that's the same, but better leave it up to influxdata.com to set the source of truth for their charts. Also this is the one I used locally when testing the influx installation.